### PR TITLE
Updated Issue Template with 6.3.1 release codepen's

### LINKED ISSUE #9825: Please Update pens! [6.3.1]
I have made the updated pens!

```markdown
<!-- Please only file either bugs or any feature request that you would like to suggest for Foundation on GitHub. If you've got a more general question about how to use Foundation, we can help you on the Foundation Forum: http://foundation.zurb.com/forum -->

#### How to reproduce this bug:

1. Step one
2. Step two
3. Step three

#### What should happen:

#### What happened instead:

#### Test case:

<!-- Give us a link to a CodePen or JSFiddle that recreates the issue. -->

- [CodePen with Foundation 6.3.0 and MotionUI](http://codepen.io/IamManchanda/pen/vgemXR)
- [CodePen with Foundation 6.3.0, Flexbox grid and MotionUI](http://codepen.io/IamManchanda/pen/xgXdRq)

- [CodePen with Foundation 6.3.1 and MotionUI](http://codepen.io/IamManchanda/pen/LWGZxR)
- [CodePen with Foundation 6.3.1, Flexbox grid and MotionUI](http://codepen.io/IamManchanda/pen/zZrBEv)
```

cc / @kball @rafibomb

### PR DESCRIPTION
**Pull request by IamManchanda**
Closes #9825 

----------------------------------------------------------------------------------------------------------

Listing below the added codepen's also so that you can review the pens if any changes required

6.3.0
- [CodePen with Foundation 6.3.0 and MotionUI](http://codepen.io/IamManchanda/pen/vgemXR)
- [CodePen with Foundation 6.3.0, Flexbox grid and MotionUI](http://codepen.io/IamManchanda/pen/xgXdRq)

6.3.1
- [CodePen with Foundation 6.3.1 and MotionUI](http://codepen.io/IamManchanda/pen/LWGZxR)
- [CodePen with Foundation 6.3.1, Flexbox grid and MotionUI](http://codepen.io/IamManchanda/pen/zZrBEv)

### CODE REVIEW

**Comment by IamManchanda** [comment]
Also a query regarding motion ui,
Last codepen's ( ncoden one's ) had motion ui version **1.1.1** ... so i kept that only for now
Shall i updated to latest version for motion ui or no ? 
Latest version is **1.2.2**

File: .github/ISSUE_TEMPLATE.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
-<!-- Please only file either bugs or any feature request that you would like to suggest for Foundation for sites on GitHub. If you've got a more general question about how to use Foundation, we can help you on the Foundation Forum: http://foundation.zurb.com/forum -->
```
Comment (Deckluhm) [inline .github/ISSUE_TEMPLATE.md]: Even if it sounds weird, the correct name is **Foundation for Sites**. That's wasn't a typo ;)

**Review by Deckluhm — CHANGES_REQUESTED** [review]

**Comment by Deckluhm** [comment]
Looking at recent [Motion UI releases](https://github.com/zurb/motion-ui/releases), I think it's perfectly ok to use the latest version here.

Another comment / question: is that really needed to have both `6.3.0` and `6.3.1`? Unless there is a good reason to have both, I think the very latest version is enough.

File: .github/ISSUE_TEMPLATE.md (hunk 1 above)
Comment (IamManchanda) [inline .github/ISSUE_TEMPLATE.md]: Ohk @Deckluhm .... moreover i by mistake did wrong branching (IamManchanda:master) so closing this and opening in another!

**Comment by IamManchanda** [comment]
No @Deckluhm 2nd last version is very important to differentiate b/w bugs crop up

**Pull request closed without merging**